packages/cli/src/commands/sandbox/sandbox_command.ts

```diff
@@ -175,6 +177,12 @@ export class SandboxCommand
           type: 'string',
           array: false,
         })
+        .option('once', {
```

Comment:
one other thought: should we make this option mutually exclusive with the file watching options like `exclude` and `dir-to-watch`?